dao/src/main/java/com/garret/movies/dao/entity/Actor.java

```diff
@@ -0,0 +1,18 @@
+package com.garret.movies.dao.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@NoArgsConstructor
```

Comment:
@Garret-vin @Data annotation already includes RequiredArgsConstructor and in your case this is empty constructor, so  you can remove NoArgsConstructor